base-compat/src/Data/Semigroup/Compat.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE CPP, NoImplicitPrelude #-}
+module Data.Semigroup.Compat (
```

Comment:
I wonder, should there be a (haddock) note that (some) symbols are only for `base-4.9`?